docs/supported_ops.md

```diff
@@ -21,6 +21,29 @@ decimals are disabled by default in the plugin, because it is supported by a sma
 number of operations presently, which can result in a lot of data movement to and
 from the GPU, slowing down processing in some cases.
 
+Result `Decimal` precision and scale follow the same rule as CPU mode in Apache Spark:
```

Comment:
This file is auto-generated so you should not edit it directly.

Reply:
Sure I just rollback the changes, and modified `TypeChecks.scala` instead.